### DISCUSSION TITLE
posts/2024-06-02-the-little-known-tricks-of-git/

### DISCUSSION THREAD

**Discussion by giscus** [Hanaasagi/blog#3] [Announcements]
# posts/2024-06-02-the-little-known-tricks-of-git/

Hanaasagi's personal blog

https://blog.dreamfever.me/posts/2024-06-02-the-little-known-tricks-of-git/

**Comment by IceCodeNew** [comment]
https://github.blog/2024-02-23-highlights-from-git-2-44/#faster-pack-generation-with-multi-pack-reuse

也想推荐一下这个配置，在 monorepo 上非常实用~
